projects/kit/utils/mask/create-auto-corrected-time-pipe.ts

```diff
@@ -1,23 +1,25 @@
 import {TuiTimeMode} from '@taiga-ui/cdk';
 import {TuiTextMaskPipeHandler} from '@taiga-ui/core';
+import {MAX_TIME_VALUES} from '@taiga-ui/kit/constants';
+import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 
 /**
  * Adjusts the entered time by omitting only suitable values for hours and minutes
  * @returns time as a string
  */
 export function tuiCreateAutoCorrectedTimePipe(
     timeMode: TuiTimeMode = 'HH:MM',
+    maxValues: Record<TuiTimeFormatParts, number> = MAX_TIME_VALUES,
 ): TuiTextMaskPipeHandler {
-    const timeFormatArray: ['HH', 'MM', 'SS', 'MS'] = ['HH', 'MM', 'SS', 'MS'];
-    const maxValue = {HH: 23, MM: 59, SS: 59, MS: 999};
+    const timeFormatArray: TuiTimeFormatParts[] = ['HH', 'MM', 'SS', 'MS'];
 
     return (conformedValue: string) => {
         const indexesOfPipedChars: number[] = [];
         const conformedValueArr = conformedValue.split('');
 
         timeFormatArray.forEach(format => {
             const position = timeMode.indexOf(format);
-            const maxFirstDigit = parseInt(maxValue[format].toString().substr(0, 1), 10);
+            const maxFirstDigit = parseInt(maxValues[format].toString().substr(0, 1), 10);
```

Comment:
`maxValues[format].`

can we change to `maxValues[format].?` for exclude null pointer in the future and create unit test where maxValues pass as `{}`

Reply:
I'll look into that